app/helpers/automated_tests_helper.rb

```diff
@@ -1,5 +1,35 @@
 # Helper methods for Testing Framework forms
 module AutomatedTestsHelper
+  # Create a repository for the test scripts and test support files
+  # if it does not exist
+  def create_test_repo(assignment)
+    # Create the automated test repository
+    unless File.exists?(MarkusConfigurator.markus_config_automated_tests_repository)
+      FileUtils.mkdir(MarkusConfigurator.markus_config_automated_tests_repository)
+    end
+
+    test_dir = File.join(MarkusConfigurator.markus_config_automated_tests_repository, assignment.short_identifier)
+    if !(File.exists?(test_dir))
```

Comment:
`File.exists?` is deprecated in favor of `File.exist?`.<br>Favor `unless` over `if` for negative conditions.